quint/test/runtime/compile.test.ts

```diff
@@ -898,6 +898,10 @@ describe('compiling specs to runtime values', () => {
               Map(2 -> 6, 3 -> 6))`,
         'true'
       )
+      assertResultAsString('Set().setOfMaps(Set(3, 5)) == Set(Map())', 'true')
+      assertResultAsString('Set().setOfMaps(Set()) == Set(Map())', 'true')
+      assertResultAsString('Set(1, 2).setOfMaps(Set()) == Set()', 'true')
```

Comment:
This makes it a bit easier to read and will also result in better error messages if it ever breaks (i.e. someone tries to remove one of the `if`s you added). The way you wrote it will result in error messages like "expected true, got false" which is not as helpful as "expected Set(), got Set(Map())".

```suggestion
      assertResultAsString('Set().setOfMaps(Set(3, 5))', 'Set(Map())')
      assertResultAsString('Set().setOfMaps(Set())', 'Set(Map())')
      assertResultAsString('Set(1, 2).setOfMaps(Set())', 'Set()')
```

Reply:
Done! 😄